stac_fastapi/api/setup.py

```diff
@@ -7,8 +7,8 @@
 
 install_requires = [
     "attrs",
-    "pydantic[dotenv]<2",
-    "stac_pydantic==2.0.*",
+    "pydantic[dotenv]>=2",
```

Comment:
I think `dotenv` was removed in pydantic 2.0  

python-dotenv is now the dependency of pedantic-settings